test/lit/non-nullable-locals.wast

```diff
@@ -0,0 +1,177 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: foreach %s %t wasm-opt -all -S -o - | filecheck %s
+
+;; Tests for the "1a" form of non-nullable locals. This will likely be the final
```

Comment:
Stray "1a" here.

Reply:
Ah, thanks, I removed "1a" from the src but not the tests... fixing now.